src/main/java/com/cleancommunity/user/MysqlUserDAO.java

```diff
@@ -71,10 +72,22 @@ public Boolean deleteUser(User user) {
 		// Returns true if query is valid and username exists
 		return this.updateQuery(sql_query);
 	}
+	public Boolean updateUser(User user) {
+		if (user instanceof Volunteer) {
+			String sql_query = String.format(
+					"UPDATE %s SET %s = %s, %s = %s, %s = %s WHERE username = \"%s\" AND password = \"%s\"",
+					TABLE_NAME, "completed", ((Volunteer)user).getProjectsCompleted(),
```

Comment:
If a value is hardcoded in the query, you should probably just indicate that in the format string instead of as an argument. i.e "completed"